### ISSUE TITLE
TensorRT Int8 speed is different between model after pytorch_quantization and model.onnx --int8 with trtexec

### ISSUE THREAD

**Issue by liuanhua110** [NVIDIA/TensorRT#1862]
trt version 8.2.1.8

**Comment by liuanhua110** [comment]
Latency of the model in trtexec add --int8 as follows:
[03/16/2022-17:51:24] [I]                                                                                                                                                                 Layer   Time (ms)   Avg. Time (ms)   Time %
[03/16/2022-17:51:24] [I]                                                                                               Reformatting CopyNode for Input Tensor 0 to Conv_0 + Relu_1 + MaxPool_2       46.09           0.0106      2.3
[03/16/2022-17:51:24] [I]                                                                                                                                           Conv_0 + Relu_1 + MaxPool_2       78.41           0.0181      3.9
[03/16/2022-17:51:24] [I]                                                                                                                                                       Conv_3 + Relu_4       46.55           0.0107      2.3
[03/16/2022-17:51:24] [I]                                                                                                                                               Conv_5 + Add_6 + Relu_7       51.50           0.0119      2.6
[03/16/2022-17:51:24] [I]                                                                                                                                                       Conv_8 + Relu_9       41.13           0.0095      2.0
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_10 + Add_11 + Relu_12       44.29           0.0102      2.2
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_13 + Relu_14       37.17           0.0086      1.9
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_16       31.14           0.0072      1.6
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_15 + Add_17 + Relu_18       43.55           0.0100      2.2
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_19 + Relu_20       45.09           0.0104      2.2
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_21 + Add_22 + Relu_23       41.89           0.0097      2.1
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_24 + Relu_25       36.62           0.0084      1.8
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_27       28.62           0.0066      1.4
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_26 + Add_28 + Relu_29       39.30           0.0091      2.0
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_30 + Relu_31       48.19           0.0111      2.4
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_32 + Add_33 + Relu_34       39.38           0.0091      2.0
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_35 + Relu_36       42.28           0.0098      2.1
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_38       29.85           0.0069      1.5
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_37 + Add_39 + Relu_40       46.92           0.0108      2.3
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_41 + Relu_42       57.26           0.0132      2.9
[03/16/2022-17:51:24] [I]                                                                                                                                            Conv_43 + Add_44 + Relu_45       45.32           0.0105      2.3
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_46 + Relu_47       27.36           0.0063      1.4
[03/16/2022-17:51:24] [I]                                                                                                                    ConvTranspose_48 + BatchNormalization_49 + Relu_50       50.27           0.0116      2.5
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_51 + Relu_52       26.13           0.0060      1.3
[03/16/2022-17:51:24] [I]                                                                                                                                                           PWN(Add_53)       22.04           0.0051      1.1
[03/16/2022-17:51:24] [I]                                                                                                                    ConvTranspose_54 + BatchNormalization_55 + Relu_56       51.16           0.0118      2.5
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_57 + Relu_58       30.65           0.0071      1.5
[03/16/2022-17:51:24] [I]                                                                                                                                                           PWN(Add_59)       23.47           0.0054      1.2
[03/16/2022-17:51:24] [I]                                                                                                                    ConvTranspose_60 + BatchNormalization_61 + Relu_62       84.82           0.0196      4.2
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_63 + Relu_64       38.30           0.0088      1.9
[03/16/2022-17:51:24] [I]                                                                                                                                                           PWN(Add_65)       34.37           0.0079      1.7
[03/16/2022-17:51:24] [I]                                                                                                                                                     Conv_66 + Relu_67       57.64           0.0133      2.9
[03/16/2022-17:51:24] [I]  Conv_68 + Relu_69 || Conv_71 + Relu_72 || Conv_74 + Relu_75 || Conv_77 + Relu_78 || Conv_80 + Relu_81 || Conv_83 + Relu_84 || Conv_86 + Relu_87 || Conv_89 + Relu_90      200.47           0.0462     10.0
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_70       39.04           0.0090      1.9
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_70       23.89           0.0055      1.2
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_73       31.27           0.0072      1.6
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_73       20.61           0.0048      1.0
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_76       31.61           0.0073      1.6
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_76       20.67           0.0048      1.0
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_79       31.89           0.0074      1.6
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_79       19.86           0.0046      1.0
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_82       32.79           0.0076      1.6
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_82       19.80           0.0046      1.0
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_85       33.18           0.0077      1.7
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_85       26.05           0.0060      1.3
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_88       33.45           0.0077      1.7
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_88       22.01           0.0051      1.1
[03/16/2022-17:51:24] [I]                                                                                                                                                               Conv_91       33.76           0.0078      1.7
[03/16/2022-17:51:24] [I]                                                                                                                  Reformatting CopyNode for Output Tensor 0 to Conv_91       20.56           0.0047      1.0
[03/16/2022-17:51:24] [I]                                                                                                                                                                 Total     2007.66           0.4630    100.0
[03/16/2022-17:51:24] [I]

**Comment by liuanhua110** [comment]
It's twice as slow, why?

**Comment by ttyio** [comment]
@liuanhua110 could you share the onnx model? (with Q/DQ and without Q/DQ) thanks!

**Comment by liuanhua110** [comment]
> @liuanhua110 could you share the onnx model? (with Q/DQ and without Q/DQ) thanks!

Sent to ur email thx~

**Comment by ttyio** [comment]
Hi all, I will close this, the pytorch_quantization is deprecated, now we moved to https://github.com/NVIDIA/TensorRT-Model-Optimizer, thanks all!
